certbot-dns-digitalocean/certbot_dns_digitalocean/dns_digitalocean_test.py

```diff
@@ -2,6 +2,7 @@
 
 import os
 import unittest
+import six
```

Comment:
I hate to nit-pick, but it also seems like `six` should be listed in the next [group of imports](https://www.python.org/dev/peps/pep-0008/#imports), instead of with standard library imports?